workflow/schemas/config.schema.yaml

```diff
@@ -1,336 +1,135 @@
 $schema: "http://json-schema.org/draft-07/schema#"
```

Comment:
this diff is too big. I read it, but cannot reliably test as a human, that all parameters are covered. I'll take your word for it ;-)